### ISSUE TITLE
Use flexbox for positioning the UI

### ISSUE THREAD

**Issue by RobertN** [sirjeppe/WebGameMaker#21]
It would be cool to use Flexbox for positioning the UI windows. The possible benefits would be to make it easier to structure how everything is placed, and would also make it easier to add additional components.

Guide:
http://www.flexboxin5.com

**Comment by sirjeppe** [comment]
Indeed! I guess we're not really targeting any older browsers without flexbox support anyway?

**Comment by RobertN** [comment]
Good point! I didn't realize it was this new: http://caniuse.com/#feat=flexbox

But yeah.. I guess the important thing is to keep the actual games working in the majority of browsers.

**Comment by sirjeppe** [comment]
Yupp :) You fix? ;)

**Comment by RobertN** [comment]
![Working](http://i.imgur.com/MvMxQ1a.gif)
